ROOTFS/etc/waggle/sanity/fatal/core_rpi.test

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+pf="RPI test"
+echo "$pf: Begin"
+
+# check core for rpi mount first
+if [ "$(df -h | grep "/media/rpi")" = "" ];
+then 
+        echo "$pf: RPI not mounted at /media/rpi FAIL"
+        exit 9
```

Comment:
Is there any significance to starting error codes at 9 for the RPi? If not, I think it'd be better to start at 1 similar to other files for uniformity.

Reply:
agreed